.env.example

```
@@ -30,3 +27,9 @@ FOAM_AWS_FINGERPRINT_BUCKET_NAME=''
 # used for creating github releases and changelog entries
 GITHUB_RELEASE_TOKEN=''
 
+
+EXPO_PUBLIC_SENTRY_DSN='' 
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The EXPO_PUBLIC_SENTRY_DSN key is duplicated

(DuplicatedKey)